### ISSUE TITLE
VMM shutting down when receiving syscall 28

### ISSUE THREAD

**Issue by luxas** [firecracker-microvm/firecracker#1037]
Is this expected or not? I get that `sys_oldfstat` is deprecated and all that, but this seems to happen pretty frequently on Firecracker v0.15.0 and I don't know what offending application I'm running in my VM that is causing this shutdown.

```
2019-03-28T13:04:59.934189258 [anonymous-instance:ERROR:vmm/src/sigsys_handler.rs:70] Shutting down VM after intercepting a bad syscall (28).
```

Thanks!

**Comment by luxas** [comment]
or has this been fixed in v0.15.2? Upgrading now and testing...

**Comment by andreeaflorescu** [comment]
Hey, can you please upgrade to Firecracker-v015.2. That version should have this problem fixed. Let us know.

**Comment by andreeaflorescu** [comment]
@luxas exactly, it should be fixed in v0.15.2.

**Comment by luxas** [comment]
Yes, seems fixed with v0.15.2. It crashed like after a minute earlier, but now it's been running for a day so I consider it working :smile:
